plugins/inventory/flightctl.py

```
@@ -0,0 +1,715 @@
+
+def _matches_label_selectors(labels: Dict[str, str], selectors: Dict[str, str]) -> bool:
+    """
+    Given a dict of labels on the device and a list of selector-dicts
+    (e.g. [{"fleet": "fleet-06"}, {"env": "prod"}]), return True
+    only if for _every_ selector dict, for every key/value in it,
+    labels[key] == value.
+    """
+    for key, expected in selectors.items():
+        if labels.get(key) != expected:
+            return False
+    return True
```

Ruff (0.8.2)

Use `return all(labels.get(key) == expected for key, expected in selectors.items())` instead of `for` loop

Replace with `return all(labels.get(key) == expected for key, expected in selectors.items())`

(SIM110)